stack/dashboards/deb/debian/postinst

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# postinst script for wazuh-dashboards
+# Copyright (C) 2021, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+set -e
+
+export NAME=wazuh-dashboards
+export CONFIG_DIR=/etc/"${NAME}"
+export INSTALLATION_DIR=/usr/share/"${NAME}"
+export LOG_DIR=/var/log/"${NAME}"
+export PID_DIR=/run/"${NAME}"
+
+
+# Setting Owner and group of own folders
+mkdir -p ${PID_DIR}
+mkdir -p ${LOG_DIR}
+chown -R "${NAME}":"${NAME}" "${LOG_DIR}"
+chown -R "${NAME}":"${NAME}" "${PID_DIR}"
+
+
+case "$1" in
+    configure)
+        setcap 'cap_net_bind_service=+ep' "${INSTALLATION_DIR}"/node/bin/node
+        if [ -f "${INSTALLATION_DIR}"/"${NAME}".restart ]; then
+        rm -f "${INSTALLATION_DIR}"/"${NAME}".restart
+            if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1 ; then
+                systemctl daemon-reload > /dev/null 2>&1
+                systemctl restart wazuh-dashboards.service > /dev/null 2>&1
+            elif command -v service > /dev/null 2>&1 && service wazuh-dashboards status 2>/dev/null | grep "running" > /dev/null 2>&1; then
```

Comment:
The grep condition will never be true

Reply:
I was able to verify that the condition is met correctly:

```
root@Debian-Buster:~# service wazuh-dashboards status | grep "running"
   Active: active (running) since Mon 2022-02-07 13:38:25 UTC; 41s ago
root@Debian-Buster:~# vim test.sh
root@Debian-Buster:~# chmod +x test.sh 
root@Debian-Buster:~# ./test.sh 
el if esta ok
root@Debian-Buster:~# cat test.sh 
#!/bin/bash

if command -v service > /dev/null 2>&1 && service wazuh-dashboards status 2>/dev/null | grep "running" > /dev/null 2>&1; then
                echo "el if esta ok"
fi

```

Not so in this line, where the condition is corrected:
https://github.com/wazuh/wazuh-packages/blob/02a29f8a7bdf3a69b9e485ca2f4d787337a8c06f/stack/dashboards/deb/debian/preinst#L69

Reply:
Done: [a4c061](https://github.com/wazuh/wazuh-packages/commit/a4c061c19a5e41836789740905f3149ae5aa90ca)